wire-library/wire-protoc/src/main/java/com/squareup/wire/protocwire/WireGenerator.kt

```diff
@@ -108,7 +111,11 @@ class WireGenerator(
       // Create a specific target and just run.
       KotlinProtocTarget().newHandler().handle(schema, CodeGeneratorResponseContext(response, sourcePaths))
     } catch (e: Throwable) {
-      response.addFile("error.log", e.stackTraceToString())
+      // Quality of life improvement.
+      val current = LocalDateTime.now()
+      val formatter = DateTimeFormatter.ofPattern("HH:mm:ss")
+      val formatted = current.format(formatter)
+      response.addFile("$formatted-error.log", e.stackTraceToString())
```

Comment:
Why catch this at all? Doesn't this get printed to stderr if you don't bother catching? I think right now, `buf generate` just silently fails when there's a problem.

The more typical alternative for protoc plugins would be to catch this at a higher level (in `Plugin` and then put this output into the `CodeGeneratorResponse.error` field. Then `buf` would see the error and report it.

Though it is also acceptable to write the error to stderr and then exit with non-zero status code, which I think is the default behavior in the JVM when there's an uncaught exception in the `main` thread.

Reply:
This is done because if we write to standard error, the protoc plug in swallows the stacktrace and prints that it doesn't understand how to decode things.

I just added a timestamp aspect here so it'll be easier to see what the error is for debugging here. I would like to remove after we're done with developing.

Reply:
Hmm, that should not be the case regarding stderr. The plugin protocol should only care about std**out**.

Also, like I mentioned, the most appropriate place to handle this is up in `Plugin`, where you can actually serialize the error message and send to `protoc` (or buf).